webpack.particle.js

```diff
@@ -150,5 +171,6 @@ module.exports = {
       // HTML from the DOM and use it as a template, then remove this line.
       vue$: 'vue/dist/vue.esm.js',
     },
+    extensions: ['.js', '.jsx', '.json', '.ts', '.tsx'],
```

Comment:
No .vue here? I need to research `extensions` more.

Reply:
Gosh darn it. Missed that one. Will add. Extensions is great because it tells webpack which extensions are resolvable if they are omitted in the import filepath.

Reply:
Updated.